Core/scripts/dirac-install.py

```diff
@@ -981,6 +981,21 @@ def fixBuildPaths():
   except:
     pass
 
+def fixPythonShebang():
+  """
+  Some scripts (like the gfal2 scripts) come with a shebang pointing to the system python.
+  We replace it with the environment one
+ """
+
+  binaryPath = os.path.join( cliParams.targetPath, cliParams.platform )
+  try:
+    replaceCmd = "grep -rIl '#!/usr/bin/python' %s/bin | xargs sed -i'.org' 's:#!/usr/bin/python:#!/usr/bin/env python:g'" %  binaryPath
```

Comment:
.org stands for "I copy pasted what was in the previous function so I kept it for consistency" :-D